src/node_http2.cc

```diff
@@ -859,11 +848,8 @@ int Http2Session::OnInvalidFrame(nghttp2_session* handle,
     HandleScope scope(isolate);
     Local<Context> context = env->context();
     Context::Scope context_scope(context);
-
-    Local<Value> argv[1] = {
-      Integer::New(isolate, lib_error_code),
-    };
-    session->MakeCallback(env->error_string(), arraysize(argv), argv);
+    Local<Value> argv = Integer::New(isolate, lib_error_code);
+    session->MakeCallback(env->error_string(), 1, &argv);
```

Comment:
You may want to call this `arg`, since it’s not really a vector any longer :)

(ditto below)